.gitignore

```diff
@@ -1,4 +1,5 @@
 lib/*
 node_modules/*
 site/*
-.*
\ No newline at end of file
+.*
+scratch/*
```

Comment:
Why add scratch/ do the gitignore?

Reply:
For the same reason as acc7095397a3de1f63d18f128330e2656a792340, just to ignore the temporary dev and test files I have there.